### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Document change
Impact:
Edit view - Change in the title for 
1. Data source
2. Single record
3. Multiple modules

### CODE REVIEW

File: widget/info.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
 	"name": "funnelChart",
 	"title": "Funnel Chart",
-	"subTitle": "Funnel Chart",
+	"subTitle": "Visualizing the Progression of Data Through layers",
 	"version": "1.0.0",
 	"published_date": 1676873166,
-
 	"metadata": {
 		"description": "To display number of records in a selected module.",
```
Comment (cs-suyog-jinturkar) [inline widget/info.json]: Is this description is correcT?

File: widget/edit.html [hunk 2]
```diff
@@ -15,31 +15,27 @@ <h3 class="modal-title col-md-9">Edit Funnel Chart</h3> <button type="button" cl
         </div>
         <div
             data-ng-class="{ 'has-error': editFunnelWidgetForm.funnelModuleType.$invalid && editFunnelWidgetForm.funnelModuleType.$touched }">
-            <label for="funnelModuleType" class="control-label">Select Dataset</label><span class="text-danger">*
+            <label for="funnelModuleType" class="control-label">Select Data Source</label><span class="text-danger">*
                 <span
-                    data-uib-tooltip="Select the 'Single Module' option if all the records to be rendered are in a single record of a certain field of the module.
-                'Across Modules' Dataset lets you select the module and it displays the count of records for the module based on the filters given"
+                    data-uib-tooltip="Select the 'Single Record' option if all the records to be rendered are in a single record of a certain field of the module.
+                'Multiple Modules' Dataset lets you select the module and it displays the count of records for the module based on the filters given"
```
Comment (cs-suyog-jinturkar) [inline widget/edit.html:21]: dataset or datasource?
Multiple module or across modules?

Comment (cs-yash-patodkar) [reply widget/edit.html:21]: Multiple modules

Comment (cs-yash-patodkar) [reply widget/edit.html:21]: Data source and multiple modules is in the final draft

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**